src/test/java/com/powsybl/openloadflow/sa/OpenSecurityAnalysisTest.java

```diff
@@ -4598,4 +4598,16 @@ void testTwoGeneratorsAtSameBus() {
         assertReactivePowerEquals(-6.5, network.getGenerator("g2").getTerminal());
 
     }
+
+    @Test
+    void testDoubleBus() {
```

Comment:
assertions to be added, no assertion causes sonar issue with "blocker" severity.

Also the issue does not lie in SA but in LfNetworkLoaderImpl, test could be limited to this class in LfNetworkLoaderImplTest and removed from OpenSecurityAnalysisTest

```java
    @Test
    void issue() {
        network = DoubleBusNetworkFactory.create();
        LfNetworkParameters networkParameters = new LfNetworkParameters()
                .setBreakers(true);
        List<LfNetwork> lfNetworks = Networks.load(network, networkParameters);
        // TODO assertions
    }
```